userguide/tutorials/overdue.adoc

```diff
@@ -204,15 +229,18 @@ curl -v \
 ----
 
 
-=== Payment retries
+=== Payment Retries System Configuration
 
-Additionally, we need to configure the payment system to retry failed payments. Each time a payment is retried, the overdue system will react and adjust the state depending on the payment status.
+The scenario defined above also requires the payments to be attempted 4 times with a gap of 1,1,8 and 6 days between attempts. Thus, the payment retry system needs to be configured as per this schedule. 
```

Comment:
I think we should emphasize that payment retries should align with overdue states. So, `1 + 1 + 8` = 10 -> aligns with `WARNING` 👍  (ok to have more retries in between though). However, what about the other states, `BLOCKED` (+4) and `CANCELLATION` (+7) ?

Reply:
Is it mandatory for payment retries to be aligned with the overdue states? I was under the impression that they need not be.

Reply:
You are right that it is a choice, but typically, it makes sense to retry the payment on the *boundary* of each overdue state, because it would prevent transitioning from one state to the next if the payment gets retried successfully. 

Example, with 2 states, 'GOOD', 'CANCELED' (where all subscriptions get canceled). Let's assume 'CANCELED' is 3 days later than 'GOOD'. If you now you only retry payment every 5 days, then you would transition to 'CANCELED'  after 3 days, *without giving a chance* to the system to retry and perhaps succeed the payment, and as a result cancel all subscriptions, and losing your customer.

So, Ideally it makes sense to align those retries with overdue state boundaries -- but they could also be more retries in between, that part is totally fine.

Reply:
Thanks for the clarification, I've now modified the overview section with an additional paragraph to explain this.